content/source/docs/cloud/sentinel/examples.html.md

```diff
@@ -39,3 +38,5 @@ This page lists some example Sentinel policies. These examples are not exhaustiv
 * [Restrict virtual disk size](https://github.com/hashicorp/terraform-guides/blob/master/governance/second-generation/vmware/restrict-vm-disk-size.sentinel)
 * [Restrict VM CPU count and memory](https://github.com/hashicorp/terraform-guides/blob/master/governance/second-generation/vmware/restrict-vm-cpu-and-memory.sentinel)
 * [Enforce NFS 4.1 and Kerberos](https://github.com/hashicorp/terraform-guides/blob/master/governance/first-generation/vmware/require_nfs41_and_kerberos.sentinel)
+
+-> **Note:** We've developed a number of first-class foundational policies to work out-of-the-box with Amazon Web Services, Microsoft Azure and Google Cloud Platform. Check out the library of [Terraform Foundational Policies](https://github.com/hashicorp/terraform-foundational-policies-library) written by HashiCorp to get up and running with your next [Policy Set](https://www.terraform.io/docs/cloud/sentinel/manage-policies.html#policies-and-policy-sets).
```

Comment:
```suggestion
-> **Note:** We've developed a number of first-class foundational policies to work out-of-the-box with Amazon Web Services, Microsoft Azure and Google Cloud Platform. Check out the library of [Terraform Foundational Policies](https://github.com/hashicorp/terraform-foundational-policies-library) written by HashiCorp to get up and running with your next [Policy Set](./manage-policies.html#policies-and-policy-sets).
```

Markup style: when linking within the same large group of documents (e.g. the Terraform Cloud docs), it's best to use relative paths, which give some benefits for future-proofing and concision. (When linking between large groups, it's best to use absolute paths but omit the protocol and hostname.) For HC-internal people, you can see more info [here](https://github.com/hashicorp/engineering-docs/blob/master/writing/markdown.md#links).